src/Linux/Avalonia.LinuxFramebuffer/Output/DrmOutput.cs

```diff
@@ -37,33 +37,6 @@ public double Scaling
                 _outputOptions = options;
             Init(card, resources, connector, modeInfo);
         }
-        public DrmOutput(string path = null, bool connectorsForceProbe = false, [CanBeNull] DrmOutputOptions options = null)
```

Comment:
IMO, this should have been left as-is

Reply:
so you mean we should try open path, getResources, getConnectors and find mode on each card enumeration?